packages/ilp-spsp-payout/src/index.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+import { PayoutConnection } from './lib/PayoutConnection'
+import makePlugin = require('ilp-plugin')
```

Comment:
@andywong418 Typescript is unhappy with the module declaration in the `ilp-plugin` package. A quick fix would be to change this to `const makePlugin = require`. However, I'd suggest importing the `ilp-plugin` package in to the monorepo and converting it fully to typescript.